src/app/modules/core-bcp/core-bcp.module.ts

```diff
@@ -13,12 +13,14 @@ import { CoreConsentModalComponent } from './components/core-consent-modal/core-
 import { CaptchaModule } from 'moh-common-lib/captcha';
 import { FacilityNumberComponent } from './components/facility-number/facility-number.component';
 import { PhoneExtensionComponent } from './components/phone-extension/phone-extension.component';
+import { MultiLineContainerComponent } from './components/multi-line-container/multi-line-container.component';
 
 const exportables = [
   CoreFacilityInfoComponent,
   CorePractitionerInfoComponent,
   CoreConsentModalComponent,
   FacilityNumberComponent,
+  MultiLineContainerComponent,
```

Comment:
Which modules are using this component - this is only for change facility it should be in the component folder under change facility.  BCP-core was meant for components that were used in majority or all modules within the application.  Please move to change facility folder

Reply:
It's used by the core `ReviewContainerComponent` component. This is to support multi-line textarea inputs on the review page.